src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -59,15 +58,22 @@ const CustomFieldInputMultipleChoice = forwardRef(function CustomFieldInputMulti
     textbox: `${props.id}-autocomplete`,
     tooltip: `${props.id}-autocomplete-tooltip`,
   };
+  const refs = {
```

Comment:
As I was already touching a bunch of files, I took it upon myself to try to clean some stuff up with some latest patterns. Hopefully it reads better and isn't painful to review. :bow: